src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -248,8 +248,7 @@ private void InitializeOpenBuffers(SnapshotSpan triggerSpan)
             this.UndoManager.CreateInitialState(this.ReplacementText, _triggerView.Selection, new SnapshotSpan(triggerSpan.Snapshot, startingSpan));
             _openTextBuffers[triggerSpan.Snapshot.TextBuffer].SetReferenceSpans(SpecializedCollections.SingletonEnumerable(startingSpan.ToTextSpan()));
 
-            UpdateReferenceLocationsTask(_threadingContext.JoinableTaskFactory.RunAsync(
-                () => _renameInfo.FindRenameLocationsAsync(_options, _cancellationTokenSource.Token)));
```

Comment:
this code was a little scary to begin with, given that it was operating on mutable class state (`_options`).